datalad/interface/run_procedure.py

```diff
@@ -192,6 +259,14 @@ class RunProcedure(Interface):
             doc="""if given, all configured paths are searched for procedures
             and one result record per discovered procedure is yielded, but
             no procedure is executed"""),
+        help_proc=Parameter(
+            args=('--help-proc',),
+            action='store_true',
+            doc="""if given, get a help message for procedure NAME from config
+            setting datalad.procedures.NAME.help. [CMD: Note, that this 
```

Comment:
The CMD note is true for other options too and isn't worth mentioning IMO.

Reply:
Yes and no. While it's true for `-d` and `--discover` as well, `-d` probably is hardly ever used and it's kind of intuitive to use `--discover` only without a `spec` argument (and therefore not run into the issue). `run-procedure XXX --help-proc` however looks perfectly reasonable but might lead to a completely unexpected and non-telling failure (or even not a failure at all depending on what `XXX` does with that argument). So, if anything we should mention it for the options, too. It might not be that obvious at a first glance that the procedures name will eat up everything following, although it should be clear once you actually think about it.

Reply:
None of that convinces me it is worth mentioning, but of course not thinking it's worth mentioning also means I don't particularly care if it *is* mentioned :]